lib/SIL/InstructionUtils.cpp

```diff
@@ -324,12 +324,15 @@ bool swift::onlyAffectsRefCount(SILInstruction *user) {
   case SILInstructionKind::StrongReleaseInst:
   case SILInstructionKind::StrongRetainInst:
   case SILInstructionKind::UnmanagedAutoreleaseValueInst:
-  case SILInstructionKind::UnmanagedReleaseValueInst:
-  case SILInstructionKind::UnmanagedRetainValueInst:
-#define ALWAYS_OR_SOMETIMES_LOADABLE_CHECKED_REF_STORAGE(Name, ...) \
-  case SILInstructionKind::Name##RetainInst: \
-  case SILInstructionKind::Name##ReleaseInst: \
-  case SILInstructionKind::StrongRetain##Name##Inst:
+#define UNCHECKED_REF_STORAGE(Name, ...)                                       \
```

Comment:
I moved the Unmanaged*ValueInst under UNCHECKED_REF_STORAGE (since Unmanaged is an unchecked ref storage).